__tests__/app.test.js

```diff
@@ -176,46 +194,34 @@ describe("Articles", () => {
     });
   });
 
-  describe("GET /api/articles/:article_id/comments", () => {
-    test("404 article_id provided is not in database", () => {
+  describe("POST /api/articles/:article_id/comments", () => {
+    test("200 valid article id and username, comment added", () => {
       return request(app)
-        .get("/api/articles/999/comments")
-        .expect(404)
-        .then((result) => {
-          expect(result.body.msg).toBe("404 - Article Not Found");
-        });
-    });
-    test("404 article_id provided is not valid type", () => {
-      return request(app)
-        .get("/api/articles/INVALID/comments")
-        .expect(400)
+        .post("/api/articles/4/comments")
+        .send({
+          username: "lurker",
+          body: "Thats not news",
+        })
+        .expect(200)
         .then((result) => {
-          expect(result.body.msg).toBe("400 - Invalid Request");
+          expect(result.body.restaurant).toMatchObject({
+            comment_id: 19,
+            body: "Thats not news",
+            article_id: 4,
+            author: "lurker",
+            votes: 0,
+            created_at: expect.any(String),
+          });
```

Comment:
Need some error testing for POST comment
invalid article id 
invalid comment body
invalid comment username
incorrect path